tests/BasicSearchTests.FunctionalTests.Core/Constants.cs

```diff
@@ -6,7 +6,8 @@ namespace BasicSearchTests.FunctionalTests.Core
     public static class Constants
     {
         // Predefined Texts
-        public const string ExistentSearchString = "Newtonsoft.Json";
+        public const string TestPackageId = "BaseTestPackage";
```

Comment:
Can you ensure that this package is available in search in Dev, INT and Prod all environments. The idea behind using Newtonsoft.Json was that it was certain to be there in these environments.